docs/web/index.md

```diff
@@ -117,6 +137,9 @@ public class AuthTestApp {
 }
 ```
 
+Additional parameters that are needed for the authorization decision may be provided with custom
+[OpaInputExtensions](../../sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/auth/opa/extension/OpaInputExtension.java).
+
 ### Testing
 
 TODO
```

Comment:
@christopher-cudennec / @JoergSiebahn Do we need to add any documentation here? 
I am not sure, what else can we add here!

Reply:
We could mention the test setup for [authentication and authorization](https://github.com/SDA-SE/sda-spring-boot-commons/tree/master/sda-commons-web-testing/src/main/java/org/sdase/commons/spring/boot/web/testing/auth).
There is also the [`GoldenFileAssertions`](https://github.com/SDA-SE/sda-spring-boot-commons/blob/master/sda-commons-web-testing/src/main/java/org/sdase/commons/spring/boot/web/testing/GoldenFileAssertions.java).
The testing module in general with the aligned dependencies and the technologies/extensions we suggest is also worth to document.

But imho that should not be part of this PR. In the long term we should provide some simple examples for testing in a separate documentation or even better in the example app. For now, I would add a sentence like this:

> The [testing module](https://github.com/SDA-SE/sda-spring-boot-commons/tree/master/sda-commons-web-testing) provides aligned test dependencies including Wiremock for external APIs and JUnit extensions to mock or disable authentication and authorization.